### PR TITLE
Calculate Nonce from account number and sequence

### PR DESCRIPTION
**Pull request by webmaster128**
Those two commits are unrelated and easier to review individually.

### CODE REVIEW

File: packages/bcp/src/cosmwasmconnection.ts [hunk 1]
```diff
@@ -372,9 +372,12 @@ export class CosmWasmConnection implements BlockchainConnection {
 
     // tslint:disable-next-line: deprecation
     const accountForHeight = await this.restClient.authAccounts(senderAddress, response.height);
-    // this is technically not the proper nonce. maybe this causes issues for sig validation?
+    const accountNumber = accountForHeight.result.value.account_number;
```
Comment (ethanfrey) [inline packages/bcp/src/cosmwasmconnection.ts]: Nice. I didn't realize there was a query here already, totally makes sense to handle this properly if we have access to the info

File: packages/bcp/src/cosmwasmconnection.ts [hunk 2]
```diff
@@ -372,9 +372,12 @@ export class CosmWasmConnection implements BlockchainConnection {
 
     // tslint:disable-next-line: deprecation
     const accountForHeight = await this.restClient.authAccounts(senderAddress, response.height);
```
Comment (ethanfrey) [inline packages/bcp/src/cosmwasmconnection.ts]: What is with the deprecation linter issue?

File: packages/bcp/src/cosmwasmconnection.ts (hunk 1 above)
Comment (webmaster128) [reply packages/bcp/src/cosmwasmconnection.ts]: I still don't know how reliable the sequence is. But we don't do verification, so it is not critical. At least the code is more expressive now.

File: packages/bcp/src/cosmwasmconnection.ts (hunk 2 above)
Comment (webmaster128) [reply packages/bcp/src/cosmwasmconnection.ts]: The field `this.restClient` is marked as depreated in order to encourace using the CosmWasmClient, which is missing a few features right now.

**Pull request merged**